backend/entities/ef/PimsProperty.cs

```diff
@@ -99,6 +99,11 @@ public PimsProperty()
         public int? Pid { get; set; }
         [Column("PIN")]
         public int? Pin { get; set; }
+        [Column("FILE_NUMBER")]
+        public int? FileNumber { get; set; }
+        [Column("FILE_NUMBER_SUFFIX")]
+        [StringLength(2)]
+        public string FileNumberSuffix { get; set; }
```

Comment:
Non concecuencial changes. The previous scaffold was generated after a Alter Up. This one after a build (db-seed)